FeBuddyWinFormUI/Program.cs

```diff
@@ -22,25 +28,30 @@ static void Main()
             try
             {
                 WebHelpers.UpdateCheck();
+
+                // Check Current Program Against Github, if different ask user if they want to update.
+                CheckVersion();
```

Comment:
Was not mentioned in the commits, need to include the following in the change log. 

If the update check could not be completed, the program would just close after alerting the user. We changed this so that it will continue with the current version after alerting the user it could not preform the update check. 

Also changed the wording on the message displayed.